### PR TITLE
Brings back sanity in KH example

### PR DESCRIPTION
**Pull request by navidcy**
This PR brings back K-H example in manageable resolution...

[I accidentally pushed a high-resolution version I was running last night.... 🤦‍♂️ Moral of the story: never merge before making sure caffeine has kicked in...]

### CODE REVIEW

File: examples/kelvin_helmholtz_instability.jl [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 using Oceananigans
 
-grid = RegularCartesianGrid(size=(256, 1, 257), x=(-5, 5), y=(0, 1), z=(-5, 5),
+grid = RegularCartesianGrid(size=(64, 1, 65), x=(-5, 5), y=(0, 1), z=(-5, 5),
```
Comment (glwagner) [inline examples/kelvin_helmholtz_instability.jl:8]: Change to 64 / power of 2 or 3?

**Review by glwagner — APPROVED** [review]

File: examples/kelvin_helmholtz_instability.jl (hunk 1 above)
Comment (navidcy) [reply examples/kelvin_helmholtz_instability.jl]: How does 320cb5a look to you?

File: examples/kelvin_helmholtz_instability.jl [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 using Oceananigans
 
-grid = RegularCartesianGrid(size=(256, 1, 257), x=(-5, 5), y=(0, 1), z=(-5, 5),
+grid = RegularCartesianGrid(size=(64, 1, 64), x=(-5, 5), y=(0, 1), z=(-5, 5),
```
Comment (glwagner) [inline examples/kelvin_helmholtz_instability.jl:8]: 🎉

File: examples/kelvin_helmholtz_instability.jl [hunk 3]
```diff
@@ -35,17 +35,18 @@ B = BackgroundField(stratification, parameters=(Ri=0.1, h=1/4))
 
 using Plots, Oceananigans.Grids
 
-z = znodes(Cell, grid)
+zF = znodes(Face, grid)
+zC = znodes(Cell, grid)
```
Comment (glwagner) [inline examples/kelvin_helmholtz_instability.jl:39]: 💯

**Review by glwagner — APPROVED** [review]
We want to slowly move towards eliminating the need for Face or Cell in user's scripts... but we aren't quite there yet. This looks like the best solution for this case.

File: examples/kelvin_helmholtz_instability.jl (hunk 3 above)
Comment (navidcy) [reply examples/kelvin_helmholtz_instability.jl:39]: for plotting the use of Plots Recipes will do the job...

**Comment by navidcy** [comment]
for plotting the use of Plots Recipes will do the job...

**Comment by glwagner** [comment]
> for plotting the use of Plots Recipes will do the job...

True, if we created a field for `Ri`.

**Pull request merged**